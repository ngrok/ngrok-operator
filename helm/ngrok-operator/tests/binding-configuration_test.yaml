suite: test binding-configuration
templates:
- binding-configuration.yaml
tests:
- it: Should match snapshot
  set:
    bindings:
      enabled: true
      name: "test-1"
  asserts:
  - matchSnapshot: {}
