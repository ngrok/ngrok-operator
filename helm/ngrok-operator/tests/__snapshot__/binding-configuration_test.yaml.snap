Should match snapshot:
  1: |
    apiVersion: bindings.k8s.ngrok.com/v1alpha1
    kind: BindingConfiguration
    metadata:
      labels:
        app.kubernetes.io/component: bindings
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: ngrok-operator
        app.kubernetes.io/part-of: ngrok-operator
        app.kubernetes.io/version: 0.12.2
        helm.sh/chart: ngrok-operator-0.15.0
      name: default-configuration
      namespace: NAMESPACE
    spec:
      allowedURLs: null
      description: Created by ngrok-operator
      name: test-1
      projectedMetadata:
        annotations: {}
        labels: {}
      region: null
      tlsSecretName: default-tls
